scripts/build-retention/monitoring/alert.rules

```diff
@@ -0,0 +1,42 @@
+groups:
+- name: alert.rules
+  rules:
+  - alert: cortx-storage
+# Change all ips below to that of the mounted node.
+    expr: 100 - (100 * ((node_filesystem_avail_bytes{device="cortx-storage.colo.seagate.com:/mnt/data1", fstype="nfs4", instance="10.230.242.163:9100", job="node-exporter", mountpoint="/rootfs/mnt/cortx-storage"} )  / (node_filesystem_size_bytes{device="cortx-storage.colo.seagate.com:/mnt/data1", fstype="nfs4", instance="10.230.242.163:9100", job="node-exporter", mountpoint="/rootfs/mnt/cortx-storage"}) )) > 95
+    for: 30s
+    labels:
+      severity: Critical
+    annotations:
+      summary: "/mnt/data1/ storage is filling fast."
+      description: "/mnt/data1/ usage is {{ humanize $value}}%. Reported by instance {{ $labels.instance }} of job {{ $labels.job }}."
+
+  - alert: releases
+# Change all ips below to that of the mounted node.
+    expr: 100 - (100 * ((node_filesystem_avail_bytes{device="cortx-storage.colo.seagate.com:/mnt/data1/releases", fstype="nfs4", instance="10.230.242.163:9100", job="node-exporter", mountpoint="/rootfs/mnt/bigstorage/releases"} )  / (node_filesystem_size_bytes{device="cortx-storage.colo.seagate.com:/mnt/data1/releases", fstype="nfs4", instance="10.230.242.163:9100", job="node-exporter", mountpoint="/rootfs/mnt/bigstorage/releases"}) )) > 90
+    for: 30s
+    labels:
+      severity: Critical
+    annotations:
+      summary: "/mnt/data1/releases storage is filling fast."
+      description: "/mnt/data1/releases usage is {{ humanize $value}}%. Reported by instance {{ $labels.instance }} of job {{ $labels.job }}."
+
+  - alert: jenkins-dev
+# Change all ips below to that of the mounted node.
+    expr: 100 - (100 * ((node_filesystem_avail_bytes{device="cortx-storage.colo.seagate.com:/mnt/data1/releases/jenkins/jenkins-dev/jenkins_home-dev", fstype="nfs4", instance="10.230.242.163:9100", job="node-exporter", mountpoint="/rootfs/mnt/jenkins-dev"} )  / (node_filesystem_size_bytes{device="cortx-storage.colo.seagate.com:/mnt/data1/releases/jenkins/jenkins-dev/jenkins_home-dev", fstype="nfs4", instance="10.230.242.163:9100", job="node-exporter", mountpoint="/rootfs/mnt/jenkins-dev"}) )) > 95
```

Comment:
Should not mention IP hardcoded

Reply:
IP is reference to build-retention node where deployment is running & LCO NFS is mounted. Developer will change IP on deploying to next node. Current IP shows deployed node and should be treated as placeholder reference.